qemu/tests/cfg/virtio_fs_set_capability.cfg

```
@@ -97,3 +97,7 @@
                     cmd_set_capability = 'setcap cap_net_raw+eip ${fs_dest}/test_file'
                     cmd_get_capability = 'getcap ${fs_dest}/test_file'
                     cmd_echo_file = 'echo "hello" > ${fs_dest}/test_file'
+                    variants:
+                        - with_xattr:
+                        - without_xattr:
+                            fs_binary_extra_options += ' -o xattr'
```

```suggestion
                        - with_xattr:
                            fs_binary_extra_options += ' -o xattr'
                        - without_xattr:
```
From my understanding and the variant name, should move `fs_binary_extra_options += ' -o xattr'` into the case with_xattr, right?

---

You are right,it's my bad. I have updated accordingly. Thanks.